src/python/CMakeLists.txt

```diff
@@ -17,16 +17,13 @@ endif()
 
 pybind11_add_module(py_generate_pipeline py_generate_pipeline.cpp)
 target_link_libraries(py_generate_pipeline PRIVATE generate_pipeline_lib)
+
 install(TARGETS py_generate_pipeline LIBRARY DESTINATION . COMPONENT genai_python)
 
 # setting RPATH / LC_RPATH depending on platform
 if(LINUX)
     # to find libgenerate_pipeline_lib.so in the same folder
     set(rpaths "$ORIGIN")
-    if(DEFINED SKBUILD)
-        # in case we build pip package, we need to refer to libopenvino.so from 'openvino' package
-        list(APPEND rpaths "@ORIGIN/../../openvino/libs")
```

Comment:
why is it removed?

Reply:
It doesn't present in tokenizers, but it felt strange to have it for mac only, so I added it. Now, when I have the precommit, I tried removing and it passed.